src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -421,10 +423,12 @@ private RootModule createRootModule() {
         final RootModule rootModule;
         try {
             final Properties props = createOverridingProperties();
-            final Configuration config =
-                ConfigurationLoader.loadConfiguration(
-                    configLocation,
-                    new PropertiesExpander(props),
+            final MultiThreadModeConfiguration multiThreadModeConfiguration =
```

Comment:
Despite type named MultiThreadModeConfiguration I would prefer singleThreadModeConfiguration for variable, as in fact you have single thread mode here.